sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -56,23 +57,69 @@ export class ServiceBusClient {
    */
   private _context: ConnectionContext;
 
+  /**
+   * Creates a ServiceBusClient for the Service Bus Namespace represented in the given connection
+   * string.
+   * @param connectionString - Connection string of the form
+   * 'Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key'
+   * @param options Options to control ways to interact with the
+   * Service Bus Namespace.
+   * @returns ServiceBusClient
+   */
+  constructor(connectionString: string, options?: ServiceBusClientOptions);
+
   /**
    * Instantiates a ServiceBusClient to interact with a Service Bus Namespace.
    *
    * @constructor
-   * @param {ConnectionConfig} config - The connection configuration needed to connect to the
-   * Service Bus Namespace.
-   * @param {TokenCredential} [tokenCredential] - SharedKeyCredential object or your 
+   * @param host - The host name for the Service Bus namespace. This is likely to be similar to
+   * <yournamespace>.servicebus.windows.net
+   * @param credential - SharedKeyCredential object or your
```

Comment:
The overload does not support SharedKeyCredential. Please update the docs

Reply:
Actually I had noticed this in the Event Hubs docs and thought it was being included on purpose.

Is AAD supported in Track 2? I see Event Hubs has a sample that uses `DefaultAzureCredential` 
Is that something we'll be using in Service Bus as well with `@azure/identity` (And in this PR?)

Reply:
SharedKeyCredential will be re-introduced for Event Hubs post Track 2 GA. There are some design  decisions that need to be finalized there.

AAD is supported in Track 2. Thats what the use of `@azure/identity` provides us.

We don't want to update the samples just yet until we are closer to releasing Track 2 previews.